tidb-binlog/pump_client/client.go

```diff
@@ -420,9 +422,12 @@ func (c *PumpsClient) Close() {
 	Logger.Infof("[pumps client] is closed")
 }
 
-func isCriticalError(err error) bool {
-	// TODO: add some critical error.
-	return false
+func isRetryableError(err error) bool {
+	if strings.Contains(err.Error(), "received message larger than max") {
```

Comment:
what about it?

Reply:
the message size reach the grpc's maxRecvMsgSize, this error is not retryable, return err directly.

Reply:
`code == ResourceExhausted` is not retryable